src/main/resources/templates/index.html

```diff
@@ -28,21 +23,31 @@ <h1>Getting Started with Gradle on Heroku</h1>
     <div class="col-md-6">
       <h3><span class="glyphicon glyphicon-info-sign"></span> How this sample app works</h3>
       <ul>
-        <li>This app was deployed to Heroku, either using Git or by using <a href="https://github.com/heroku/java-getting-started">Heroku Button</a> on the repository.</li>
+        <li>This app was deployed to Heroku, either using Git or by using <a href="https://github.com/heroku/gradle-getting-started">Heroku Button</a> on the repository.</li>
 
-        <li>When Heroku received the source code, it grabbed all the dependencies in the <a href="https://github.com/heroku/java-getting-started/blob/master/pom.xml">pom.xml</a>.</li>
+        <li>When Heroku received the source code, it fetched all the dependencies in the <a href="https://github.com/heroku/gradle-getting-started/blob/master/pom.xml">pom.xml</a>, creating a slug.</li>
         <li>The platform then spins up a dyno, a lightweight container that provides an isolated environment in which the slug can be mounted and executed.</li>
         <li>You can scale your app, manage it, and deploy over <a href="https://addons.heroku.com/">150 add-on services</a>, from the Dashboard or CLI.</li>
-        <li>Check out the <a href="https://devcenter.heroku.com/articles/getting-started-with-java">Getting Started</a> guide to learn more!</li>
       </ul>
     </div>
     <div class="col-md-6">
+      <h3><span class="glyphicon glyphicon-link"></span> Next Steps</h3>
+      <ul>
+        <li>If you are following the <a href="https://devcenter.heroku.com/articles/getting-started-with-gradle">Getting Started</a> guide, then please head back to the tutorial and follow the next steps!</li>
```

Comment:
Thanks to the magic of Dev Center inconsistency, the slug for the Gradle tutorial is actually `getting-started-with-gradle-on-heroku`, so we should update this URL here and anywhere else it appears in the repo

Reply:
fixed in 29982c6